graylog2-server/src/main/java/org/graylog/events/rest/EventDefinitionsResource.java

```diff
@@ -469,6 +472,21 @@ public ValidationResult validate(@ApiParam(name = "JSON body", required = true)
         return validationResult;
     }
 
+    @POST
+    @Path("/validate_cron")
```

Comment:
Let's put this into a sub-path to have a namespace for future validations.

```suggestion
    @Path("/validate/cron_expression")
```